auth/v1.js

```diff
@@ -134,14 +128,14 @@ api.declare({
 }, async function(req, res) {
   let clientId = req.params.clientId;
 
-  // Load client
-  let client = await this.Client.load({clientId}, true);
+  // Load client from the resolver's cache
+  let client = this.resolver.jsonClient(clientId);
```

Comment:
Hm, last we talked design, we agreed it was safe to do prefix filtering for clients because we weren't doing table scans.  With 1000's of clients, are you still comfortable with the extra load?